pkg/controller/servicebinding/reconciler_test.go

```diff
@@ -189,17 +189,16 @@ func TestReconcilerReconcileUsingSecret(t *testing.T) {
 		require.Equal(t, 1, len(containers))
 		require.Equal(t, 1, len(containers[0].EnvFrom))
 		require.NotNil(t, containers[0].EnvFrom[0].SecretRef)
-		require.Equal(t, reconcilerName, containers[0].EnvFrom[0].SecretRef.Name)
+		require.Contains(t, containers[0].EnvFrom[0].SecretRef.Name, reconcilerName)
```

Comment:
Please include the checksum in the validation; it should be thought as a contract that the secret is named as expected.

`Contains` might match different resources, since it has no boundaries.

Reply:
Done here cfb3675